Makefile

```diff
@@ -139,6 +139,7 @@ clean:
 		cd arc/; make clean; \
 	fi
 	make -f Makefile.linux clean
+	@rm -rf $(JERRY_BASE)/build
```

Comment:
It would be better for this to be BOARD-specific, but to do this nuclear option in 'make pristine'.

However, I see that line is already in the above section... so why isn't that working now? Also why is it conditional under ZEPHYR_SDK_INSTALL_DIR? I don't think that makes sense.